src/app/collection-page/collection-page.component.ts

```diff
@@ -121,4 +122,18 @@ export class CollectionPage implements OnInit, OnDestroy {
   // private updateSearchInRes(value: boolean): void{
   //  this.searchInResults = value;
   // }
+
+  public skipToFilterSec(): void{
```

Comment:
also here

Reply:
@willschoener  updated in the next commit `dd77c08`.